kokoro/continuous.sh

```diff
@@ -12,7 +9,7 @@ gcloud components install docker-credential-gcr
 export PATH=$PATH:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin/
 
 # docker-credential-gcr uses GOOGLE_APPLICATION_CREDENTIALS as the credentials key file
-export GOOGLE_APPLICATION_CREDENTIALS=$(pwd)/keyfile.json
+export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_KEYSTORE_DIR}/72743_jib_integration_testing_key
```

Comment:
Wouldn't this mean that the key is still in the environment variable `GOOGLE_APPLICATION_CREDENTIALS`?

Reply:
The difference is that the contents of the key were in `JIB_INTEGRATION_TESTING_KEY`, and those contents were being logged in plain text. `GOOGLE_APPLICATION_CREDENTIALS` just holds a path.

Reply:
Kk, we'll just have to make sure we don't ever log the contents of that file in our build logs.

Reply:
I think my concern is that some malicious PR will expose this and someone accidentally triggers a build for it.

Reply:
Yeah, anyone could do `echo $JIB_INTEGRATION_TESTING_KEY` to get the contents of the service account key, so this PR doesn't make things worse. That's why we should never auto-trigger Kokoro builds for an external contributor, which is the current setting.

What I did for CT4E is to use the service key of the GCE machine where Kokoro runs, but unfortunately this required using a dedicated machine pool that we maintain ourselves.

Reply:
@chanseokoh 

> That's why we should never auto-trigger Kokoro builds for an external contributor, which is the current setting.

How do you configure this?

Reply:
Chatted with with @etanshaul.